filter_plugins/oo_filters.py

```diff
@@ -994,6 +994,60 @@ def oo_random_word(length, source='abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRS
     return ''.join(random.choice(source) for i in range(length))
 
 
+def norm(value, prefix_map):
+    ''' Normalize value based on the prefix_map '''
+    value = value.strip(' ')
+    match = re.search('([0-9.]*)(.*)', value)
+    val = match.group(1)
+    unit = match.group(2).strip(' ')
+    if unit not in prefix_map:
```

Comment:
Not that this is performance sensitive, but using try/except is better than a double test:

```
try:
    return float(val) * prefix_map[unit]
except KeyError:
    raise ValueError('Unit ' + unit + ' not supported')
```